keyboards/newgame40/keymaps/default/keymap.c

```diff
@@ -0,0 +1,213 @@
+  * |------+------+------+------+------+------|------+------+------+------|
+  * |   A  |   R  |   S  |   T  |   D  |   H  |   N  |   E  |   I  |   O  |
+  * |------+------+------+------+------+------+-------------+------+------|
+  * |   Z  |   X  |   C  |   V  |   B  |   K  |   M  |   ,  |   .  |   /  |
+  * |-------------+------+------+------+------+------+------+------+------'
+  * |Shift |  GUI |Lower | Ctrl |Space | Bksp |Enter |Raise | Left |Right |
+  * `---------------------------------------------------------------------'
+  */
+ [_COLEMAK] = LAYOUT( \
+   KC_Q,     KC_W,     KC_F,      KC_P,      KC_G,      KC_J,      KC_L,      KC_U,      KC_Y,      KC_MINS,  \
+   KC_A,     KC_R,     KC_S,      KC_T,      KC_D,      KC_H,      KC_N,      KC_E,      KC_I,      KC_O,     \
+   KC_Z,     KC_X,     KC_C,      KC_V,      KC_B,      KC_K,      KC_M,      KC_COMM,   KC_DOT,    KC_SLSH,  \
+   KC_LSFT, KC_LGUI, LT(_LOWER,KC_LANG2),  KC_LCTL, KC_SPC,  KC_BSPC, KC_ENT, LT(_RAISE,KC_LANG1),  KC_LEFT, KC_RGHT\
+ ),
+
+ /* Dvorak
+  * ,---------------------------------------------------------------------.
+  * |   "  |   ,  |   .  |   P  |   Y  |   F  |   G  |   C  |   R  |   L  |
+  * |------+------+------+------+------+------|------+------+------+------|
+  * |   A  |   O  |   E  |   U  |   I  |   D  |   H  |   T  |   N  |   S  |
+  * |------+------+------+------+------+------+-------------+------+------|
+  * |   ;  |   Q  |   J  |   K  |   X  |   B  |   M  |   W  |   V  |   Z  |
+  * |-------------+------+------+------+------+------+------+------+------'
+  * |Shift |  GUI |Lower | Ctrl |Space | Bksp |Enter |Raise | Left |Right |
+  * `---------------------------------------------------------------------'
+  */
+ [_DVORAK] = LAYOUT( \
+   KC_QUOT,   KC_COMM,  KC_DOT,    KC_P,      KC_Y,       KC_F,     KC_G,      KC_C,     KC_R,      KC_L, \
+   KC_A,      KC_O,     KC_E,      KC_U,      KC_I,       KC_D,     KC_H,      KC_T,     KC_N,      KC_S, \
+   KC_SCLN,   KC_Q,     KC_J,      KC_K,      KC_X,       KC_B,     KC_M,      KC_W,     KC_V,      KC_Z, \
+   KC_LSFT, KC_LGUI, LT(_LOWER,KC_LANG2),  KC_LCTL, KC_SPC,  KC_BSPC, KC_ENT, LT(_RAISE,KC_LANG1),  KC_LEFT, KC_RGHT\
+ ),
+
+ /* Lower
+  * ,---------------------------------------------------------------------.
+  * |  Esc |      |      |      |      |      |      |      |      |      |
+  * |------+------+------+------+------+------|------+------+------+------|
+  * |      |      |      |      |      |      |      |      |      |      |
+  * |------+------+------+------+------+------+-------------+------+------|
+  * |  F13 |  F14 |  F15 |  F16 |  F17 |  F18 | F19  | F20  | F21  | F22  |
+  * |-------------+------+------+------+------+------+------+------+------'
+  * |  F1  | F2   |      | F3   |  F4  |  F5  |  F6  |      |  F7 |   F8  |
+  * `---------------------------------------------------------------------'
+  */
+ [_LOWER] = LAYOUT( \
```

Comment:
If you're using `LAYOUTS = ortho_4x10`, then you should use that here: 

```suggestion
 [_LOWER] = LAYOUT_ortho_4x10( \
```